src/lambdas/dispatcher/src/lambda_function.py

```diff
@@ -34,6 +40,9 @@ def handler(event, context):
         if isinstance(command, AnalysisCommand):
             cast(AnalysisCommand, command)
             jobs += _analysis(command, client)
+        elif isinstance(command, RasterVersionUpdateCommand):
+            cast(RasterVersionUpdateCommand, command)
+            jobs += _raster_version_update(command)
```

Comment:
This entire block also calls for a refactor.
I would create a `get_jobs` method and override it for each Command type.

This way, you would only need to call `jobs = command.get_jobs()`